<launch>

  <node name="noisy_pose_guess_gen" pkg="state_estimation" type="noisy_pose_guess_generator" respawn="false" output="screen" clear_params="true">
    <param name="xyz" type="string" value="-0.07 0.28 0.05" />
    <param name="rpy" type="string" value="-90 0 0" />
    <param name="pos_noise" type="double" value="0.05" />
    <param name="rot_noise" type="double" value="20" />
    <param name="output_file" type="string" value="$(find state_estimation)/calibration/vision_pose_guess.yaml" />
  </node>

</launch>
